docs/guides/deployment.md

```diff
@@ -1,2 +1,158 @@
 # Deployment
+This section contains guides and advices related to Raccoon deployment.
+## Standalone
+This section contains standalone deployment options for Raccoon server. You can use this deployment method on your local machine to get started with Raccoon.
+### Local Machine
+**Prerequisite**
+- GO 1.14 or higher installed
+- Unix based machine
 
+**Run The Executable**
+Before you can run the server on your local machine. First, you need to compile it using the following command.
+```sh
+$ git clone https://github.com/odpf/raccoon
+$ cd raccoon
+$ make
+```
+You can find the executable on `./out/raccoon`.
+```sh
+# Run the executable
+$ ./out/raccoon
+```
+
+**Configuration**
+You can have the [configuration](https://odpf.gitbook.io/raccoon/reference/configurations) set either in `.env` file where you run the executable or export it as env variable.
+
+### Docker
```

Comment:
Instead of Docker you could call this section as `Deploying Raccoon in kubernetes` and arrange the sub section/text in such a way that the user of Raccoon knows how to deploy in kubernetes.